tardis/montecarlo/montecarlo_numba/base.py

```diff
@@ -196,9 +196,9 @@ def montecarlo_main_loop(
         )
 
     # Configuring the Tracking for R_Packets
-    tracked_rpackets = List()
+    tracked_rpackets_collection = List()
```

Comment:
None of them are collections. Name it rpacket_trackers or something

Reply:
Reasoning behind naming it a collection is that it is a collection of `RPacketTracker` class instances. It also makes sense that `rpacket_trackers` is more apt for the general information that this variable may impart. I will change it accordingly :)